tide_core.links.menu.yml

```
@@ -0,0 +1,6 @@
+tide_core.content_summary_page:
+  title: Content Summary
+  description: 'improved contents view page'
```

```suggestion
  description: 'Content listing with additional filters for site, sub-site and workflow state.'
```

Providing more specific detail of the function this page provides.